src/components/Tabs.vue

```diff
@@ -56,6 +56,9 @@
     },
     methods: {
       select_tab (i) {
+        // provide parent component with selected tab index
+        this.$emit('selected-index', i)
+
         this.selectedIndex = i
```

Comment:
Very minor, but even for something simple like this I think a more descriptive variable name than `i` would be better.